src/lib/config_v2/testdata/config_API_valid.yml

```
@@ -0,0 +1,50 @@
+apis:
+    - name: route-name-one
+      every: 3h0m0s
+      domain_or_ip: www.some-url-for-one.com
+      route: /api/v1/test
+      method: get
```

```suggestion
      method: GET
```
Lets have some combinations of small and capitals.